inc/settings.php

```diff
@@ -10,6 +10,10 @@ class Optml_Settings {
 	const FILTER_FILENAME = 'filename';
 	const FILTER_TYPE_LAZYLOAD = 'lazyload';
 	const FILTER_TYPE_OPTIMIZE = 'optimize';
+
+	const WATCHER_TYPE_LAZYLOAD = 'lazyload';
+	const WATCH_CLASS = 'class';
```

Comment:
do we still need `WATCH_CLASS` and  `WATCH_ID`, i'm not able to find their usage